### ISSUE TITLE
webdriver_manager v3.6.0 and v3.6.1 does not work with udemy_enroller

### ISSUE THREAD

**Issue by nottrif** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#374]
I tried installing the script with v3.6.1 and v3.6.1 but the script didn't work for me.
ModuleNotFoundError: No module named 'webdriver_manager.utils' this was the error I got.
I had to downgrade to v3.5.4 and it worked like a charm.
PS: webdriver_manager v3.6.0 and v3.6.1 are fairly new. (released 40 minutes ago from the time of my installation)

**Comment by notjedi** [comment]
changing `webdriver_manager.utils` to `webdriver_manager.core.utils` should work

**Comment by pinguluk** [comment]
> changing `webdriver_manager.utils` to `webdriver_manager.core.utils` should work
> 
> EDIT: this is a good first issue, so you can prolly try to do a PR @nottrif

fixed my issue

**Comment by cullzie** [comment]
Ye this is also to do with how dependencies are not tied to a specific version. I'll make changes to how the dependencies are resolved. Thanks @nottrif

**Comment by notjedi** [comment]
yeah, resolving to a specific version is always a good practice. for example, `webdriver_manager==3.5.4` should have worked fine.

**Comment by cullzie** [comment]
Ye its playing the a risky game but on the flip side there are no code changes required to get updates to external dependencies :laughing:

**Comment by cullzie** [comment]
@pinguluk Latest version 4.1.1 should resolve this.

**Comment by s0j0urn** [comment]
@cullzie I still get an error 

```
driver_manager.py", line 4, in <module>
    from webdriver_manager.core.utils import ChromeType
ModuleNotFoundError: No module named 'webdriver_manager.core'
```

**Comment by notjedi** [comment]
just upgrade `webdriver_manager` using `pip install --upgrade webdriver-manager`.

**Comment by s0j0urn** [comment]
@notjedi Thank you. I did upgrade and then started hitting another exception

```
udemy_enroller.py
Loading existing settings
Exception in redeem courses: __init__() got an unexpected keyword argument 'ecdhCurve'
```

**Comment by notjedi** [comment]
can you paste the exact exception? with the line number? i only see the `ecdhCurve` keyword [here](https://github.com/aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE/blob/master/udemy_enroller/udemy_rest.py#L112) in the whole project

**Comment by s0j0urn** [comment]
Fixed it. I had to reinstall the dependencies.
